### PR TITLE
feat(doc) refactoring crypto tracker blog post

### PR DESCRIPTION
**Pull request by amyshwang**

### CODE REVIEW

File: blog/2023-01-12-realtime-crypto-tracker-with-kafka-and-questdb.mdx [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+With the new connector, we can just send the payload directly:
+
+```json
+{
+ "timestamp": int(time.time() * 1000),
+ "currency": raw_data['data']['base'],
+ "amount": float(raw_data['data']['amount'])
+}
+```jason
```
Comment (jerrinot) [inline blog/2023-01-12-realtime-crypto-tracker-with-kafka-and-questdb.mdx:172]: typo?

File: blog/2023-01-12-realtime-crypto-tracker-with-kafka-and-questdb.mdx [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+  - kafka
+  - pandas
+  - python
+tags: [tutorial, python, pandas, kafka, crypto]
+image: /img/blog/2023-01-12/logos.png
+slug: realtime-crypto-tracker-with-questdb-kafka-connector
+---
+
+[Yitaek Hwang](https://yitaek.medium.com/) introduces a more performant change
+data capture (CDC) setup to stream data from Kafka to QuestDB leveraging the
```
Comment (jerrinot) [inline blog/2023-01-12-realtime-crypto-tracker-with-kafka-and-questdb.mdx:22]: this is not about CDC at all, is it?

File: blog/2023-01-12-realtime-crypto-tracker-with-kafka-and-questdb.mdx [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+
+1. Required explicit schema information for messages
+
+To address these challenges, the QuestDB team recently released the official
+[QuestDB Kafka connector](https://questdb.io/docs/third-party-tools/kafka/questdb-kafka/).
+Underneath the hood, the new connector uses InfluxDB line protocol (ILP), which
+has superior ingestion performance over the JDBC driver utilizing the PostgreSQL
+wire and simplifies schema management. So I decided to give this a spin and
+apply it to my original project:
+[**GitHub - Yitaek/kafka-crypto-questdb: Using Kafka to track cryptocurrency price trends** *This repo polls Coinbase API for various cryptocurrency prices and uses Kafka to calculate moving average prices and…*github.com](https://github.com/Yitaek/kafka-crypto-questdb)
```
Comment (jerrinot) [inline blog/2023-01-12-realtime-crypto-tracker-with-kafka-and-questdb.mdx:55]: bad formatting? it renders like

**Review by jerrinot — APPROVED** [review]
a few minor things, but it looks good.

perhaps consider adding a notice box to the old article. something like: "This is an old article, kept for archiving purposes. New projects should use the QuetDB connector instead of the JDBC connector (link to this article)". So people who stumble upon the old post (Google, StackOverflow links, etc) won't use the JDBC driver.

File: blog/2023-01-12-realtime-crypto-tracker-with-kafka-and-questdb.mdx (hunk 1 above)
Comment (argshook) [reply blog/2023-01-12-realtime-crypto-tracker-with-kafka-and-questdb.mdx:172]:
```suggestion
```json
```

typo, thanks!

File: blog/2023-01-12-realtime-crypto-tracker-with-kafka-and-questdb.mdx (hunk 2 above)
Comment (amyshwang) [reply blog/2023-01-12-realtime-crypto-tracker-with-kafka-and-questdb.mdx:22]: Will update but this was what he wrote...

**Review by jerrinot — APPROVED** [review]

**Pull request merged**